### PR TITLE
Add man page for OpenRCT2

### PR DESCRIPTION
**Pull request by IntelOrca**
Not sure if I have done this right...

### CODE REVIEW

**Comment by janisozaur** [comment]
Oh _man_…

**Comment by marijnvdwerf** [comment]
Wouldn't it make more sense to make the built executable generate parts of it?

**Comment by IntelOrca** [comment]
> Wouldn't it make more sense to make the built executable generate parts of it?

What do you mean?

**Comment by marijnvdwerf** [comment]
Most of this stuff seems to match `openrct2 --ha`, albeit formatted in a different way. Wouldn't it be possible in some way to let `./openrct2 --man` (or something) generate the output in manpage-form? That way you wouldn't need to write things twice, or have possibly outdated man pages.

**Comment by IntelOrca** [comment]
Maybe, if you or someone embarks on that, it can replace that static file we have now.

**Comment by Gymnasiast** [comment]
Can we just merge this (provided it's well-formed) and not get bogged down on what might be marginally better _again_? Create a new issue if you really feel it's worth the effort, but please, don't let perfect stand in the way of 'good enough'.

**Comment by IntelOrca** [comment]
@Gymnasiast I just need confirmation that this is correct. Is it?

**Comment by Gymnasiast** [comment]
I think so, but then @janisozaur probably knows more about this than I do. But seeing as he took the opportunity to comment and only posted a joke, it appears he didn't find anything wrong.

File: distribution/man/openrct2.6 [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+Path to the RollerCoaster Tycoon 2 data directory (containing
+.Pa data/g1.dat )
+
+.Sh EXAMPLES
+.Bl -tag -width "openrct2 https://openrct2.website/files/SnowyPark.sv6 "
+.It openrct2 ./my_park.sv6
+Open a saved park.
+.It openrct2 https://openrct2.website/files/SnowyPark.sv6
+Download and open a saved park.
+.It host ./my_park.sv6 --port 11753 --headless
```
Comment (janisozaur) [inline distribution/man/openrct2.6]: missing `openrct2`

File: distribution/man/openrct2.6 [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+Show information about
+.Nm .
+
+.It Fl -verbose
+Print diagnostic information to stdout.
+
+.It Fl -headless
+run
+.Nm
+as a background process.
```
Comment (janisozaur) [inline distribution/man/openrct2.6]: That's not true, though. "background" would be understood as forking into a daemon, which we don't do and still hold control of TTY. Something along the lines of "run without spawning graphical window" would be more appropriate.

Comment (Gymnasiast) [inline distribution/man/openrct2.6]: That (or a similar phrasing) would probably be better, as it is not unthinkable that OpenRCT2 would be capable of running as a daemon. It'd be more precise and less ambiguous.

Comment (IntelOrca) [inline distribution/man/openrct2.6]: I obviously got confused with what it says for OpenTTD which does says something like fork it.

**Pull request merged**